ui/values/dimensions.go

```diff
@@ -90,6 +90,6 @@ var (
 	TextSize34   = unit.Sp(34)
 	TextSize32   = unit.Sp(32)
 
-	AppWidth  = unit.Sp(800)
-	AppHeight = unit.Sp(600)
+	AppWidth  = unit.Sp(428)
```

Comment:
The reason we had this minimum with was to protect some vital layouts. example are, 

<img width="436" alt="image" src="https://user-images.githubusercontent.com/27733432/170943502-d8bbb93c-f803-4e23-8042-8f0d969b377b.png">

<img width="430" alt="image" src="https://user-images.githubusercontent.com/27733432/170943603-0bfb4526-14d1-4ba8-99de-e674a3f3d4e4.png">

<img width="430" alt="image" src="https://user-images.githubusercontent.com/27733432/170943768-ad2eb63a-78c3-466d-bef4-760972d497d5.png">

I understand that this PR is only a part to a larger PR.. however, this change is going to break the current ui.

What we can do, is to revert this change and while testing your PRs and subsequent PRs, you can modify.

Reply:
okay noted, thanks for pointing it out